### DISCUSSION TITLE
Closed onboarding PRs are recreated each time

### DISCUSSION THREAD

**Discussion by dodys** [renovatebot/renovate#26133] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Closing renovate's PR is not working and is spamming out of control since last friday.
See:
https://github.com/canonical/sec-cvescan/pulls?q=is%3Apr+author%3Aapp%2Frenovate+is%3Aclosed

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment, marked as answer]
- #26090

**Reply by rarkins** [reply]
Workaround: Rename one of the PRs to "Configure Renovate"
